exercises/exercise_1/README.md

```diff
@@ -1,336 +1,197 @@
-Creating words_db_1_e689fccd4ad3    ... done
-```
 
-As you can see, the `deploy` section of the `words` service is ignored, so there's only one replica.
-
-* **Check the logs** using `docker-compose logs`. It will print the logs for each service and replicas.
-```sh
-$ docker-compose logs
-Attaching to workshop_web_1_8aa6fd5ddef2, workshop_words_1_adba85d19515, workshop_db_1_d44dc3005667
-db_1_d44dc3005667 | The files belonging to this database system will be owned by user "postgres".
-db_1_d44dc3005667 | This user must also own the server process.
-db_1_d44dc3005667 |
-db_1_d44dc3005667 | The database cluster will be initialized with locale "en_US.utf8".
-db_1_d44dc3005667 | The default database encoding has accordingly been set to "UTF8".
-db_1_d44dc3005667 | The default text search configuration will be set to "english".
-...
-```
 
-* **Scale** the words service to 3 replicas using the command `docker-compose scale`
-```sh
-$ docker-compose scale words=3
-Starting words_words_1_dee4bda4d1be ... done
-Creating words_words_2_bb72fab8e9bc ... done
-Creating words_words_3_1df7aa0cc47b ... done
-```
+### Persisting our Database's Data
 
-* **Shutdown the services** using `docker-compose down`
-```sh
-$ docker-compose down
-docker-compose down
-Stopping words_words_3_3a79e54a6e1e ... done
-Stopping words_words_2_e02d3e977440 ... done
-Stopping words_db_1_d542f458574a    ... done
-Stopping words_web_1_f090b73d9c1f   ... done
-Stopping words_words_1_dee4bda4d1be ... done
-Removing words_words_3_3a79e54a6e1e ... done
-Removing words_words_2_e02d3e977440 ... done
-Removing words_db_1_d542f458574a    ... done
-Removing words_web_1_f090b73d9c1f   ... done
-Removing words_words_1_dee4bda4d1be ... done
-Removing network words_defaul
-```
+Databases are pretty important and it's usually preferred to keep the data longer than a single container restart. As such, we need to store the data in a volume. 
 
-**Summary**:
-- `docker-compose` can quickly pull, deploy and remove your services.
-- You can scale any service, but the `deploy` section of the compose file is ignored.
+In this case, we don't care _where_ the data is stored, as long as it's there every time we start the container. This is a good candidate for using a named volume. 
 
-#### `docker-compose` scale vs `docker stack` scale
+1. We first need to declare a new volume in the `volumes` section. Let's just name it `db-data` and let it use the default `local` driver.
 
-* **Re-up** your first `docker-compose.yml` file with `docker-compose up -d`
-* Try to **scale** the `hello` service to **3**
-```sh
-$ docker-compose scale hello=3
-WARNING: The "hello" service specifies a port on the host. If multiple containers for this service are created on a single host, the portwill clash.
-Starting workshop_hello_1_e715af2868fd ... done
-Creating workshop_hello_2_820416f01715 ... error
-Creating workshop_hello_3_d28c00b40f0c ... error
+    <details>
+      <summary>Solution</summary>
+    ```yaml
```

Comment:
I think you are missing an empty line after `<summary>`, the yaml is not well presented.